packages/contracts/tasks/helpers/benchmarks.ts

```diff
@@ -38,9 +38,9 @@ export async function publishBatch(
   let optimalBatchSize = batchSize;
 
   while (optimalBatchSize > 0) {
+    console.log(`Attempting to publish batch of ${optimalBatchSize} messages`);
```

Comment:
was this for debugging while testing or something you think we should keep?

Reply:
I think we can remove it, I used to test when the transaction wasn't picked, but isn't really useful.